mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 DECORATION Lnet/minecraft/class_3614;
 	FIELD field_15925 CACTUS Lnet/minecraft/class_3614;
-	FIELD field_15926 SEAGRASS Lnet/minecraft/class_3614;
+	FIELD field_15926 REPLACEABLE_AQUATIC_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15927 STRUCTURE_VOID Lnet/minecraft/class_3614;
-	FIELD field_15928 PACKED_ICE Lnet/minecraft/class_3614;
+	FIELD field_15928 DENSE_ICE Lnet/minecraft/class_3614;
 	FIELD field_15929 liquid Z
 	FIELD field_15930 EGG Lnet/minecraft/class_3614;
 	FIELD field_15931 WOOL Lnet/minecraft/class_3614;
 	FIELD field_15932 WOOD Lnet/minecraft/class_3614;
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
 	FIELD field_15937 CAKE Lnet/minecraft/class_3614;
-	FIELD field_15938 BAMBOO_SAPLING Lnet/minecraft/class_3614;
+	FIELD field_15938 BAMBOO_SHOOT Lnet/minecraft/class_3614;
 	FIELD field_15939 blocksLight Z
 	FIELD field_15940 blocksMovement Z
-	FIELD field_15941 EARTH Lnet/minecraft/class_3614;
+	FIELD field_15941 TOPSOIL Lnet/minecraft/class_3614;
 	FIELD field_15942 GLASS Lnet/minecraft/class_3614;
 	FIELD field_15943 FIRE Lnet/minecraft/class_3614;
 	FIELD field_15944 color Lnet/minecraft/class_3620;
-	FIELD field_15945 ORGANIC Lnet/minecraft/class_3614;
+	FIELD field_15945 FODDER Lnet/minecraft/class_3614;
```

Comment:
```suggestion
	FIELD field_15945 ORGANIC Lnet/minecraft/class_3614;
```
I wouldn't call mycelium or dried kelp "fodder".

Reply:
Dried kelp can be used for food, and I'm ignoring the odd man out of mycelium.
The goal is to make the name clearer as to what blocks it represents. Suggestions are welcome.

Reply:
compared to clay, this one is like raw biomass/living biomass while clay/honey block etc are non-living bio-derived stuff

Reply:
the fodder is that most of the blocks in it are a food source, or storage of one
I'm implying that mooshrooms eat mycelium with that name, but fodder encompasses most of the blocks in that material
since organic is far too broad

Reply:
`ORGANIC` -> `BIO` or `BIOMASS`

I'm not sure what fodder means, other than something you throw dock for pigs to eat.

Reply:
Fodder is the storage of food for later use. This or a synonym would function well. Biomass is far too general for this group.
![image](https://user-images.githubusercontent.com/4873274/76987909-512a1500-691a-11ea-9183-9a8357d59956.png)

Reply:
Not an improvement because it loses accuracy for certain of the blocks with the material

Reply:
FODDER -> SOLID_ORGANIC to better encompass all the blocks that use it, and to narrow down the blocks it applies to